src/backend/columnar/columnar_customscan.c

```diff
@@ -121,10 +121,8 @@ static void ColumnarScan_ExplainCustomScan(CustomScanState *node, List *ancestor
 static const char * ColumnarPushdownClausesStr(List *context, List *clauses);
 static const char * ColumnarProjectedColumnsStr(List *context,
 												List *projectedColumns);
-#if PG_VERSION_NUM >= 130000
```

Comment:
it seems we have 1 more such place to remove ifdef?
```
grep -Ri "PG_VERSION_13" src/backend/distributed/
src/backend/distributed//transaction/relation_access_tracking.c:#if PG_VERSION_NUM >= PG_VERSION_13
```

Reply:
PG_VERSION_12 can also be removed from `src/include/distributed/pg_version_constants.h`

Reply:
There are also some comments throughout our code and tests that indicate certain things only exist because of PG11/PG12 support. Some examples (not all of these should have the code changed, but at least the comment can be changed because it applies to all supported PG versions):
```
❯ rg pg12 --ignore-case -A 2 -B 2 **.c
src/backend/distributed/planner/function_call_delegation.c
171-
172-        /*
173:         * In pg12's planning phase empty FROMs are represented with an RTE_RESULT.
174-         * When we arrive here, standard_planner has already been called which calls
175-         * replace_empty_jointree() which replaces empty fromlist with a list of

src/backend/distributed/metadata/metadata_sync.c
3808- * be used to obtain the OID of a type on a different node when included
3809- * in a query string. Currently this is a sublink because regcollation type
3810: * is not available in PG12.
3811- */
3812-static char *

src/backend/distributed/deparser/citus_ruleutils.c
527-
528-    /*
529:     * Add table access methods for pg12 and higher when the table is configured with an
530-     * access method
531-     */

src/backend/distributed/commands/function.c
920-/*
921- * GetAggregateDDLCommand returns a string for creating an aggregate.
922: * CREATE OR REPLACE AGGREGATE was only introduced in pg12,
923- * so a second parameter useCreateOrReplace signals whether to
924- * to create a plain CREATE AGGREGATE or not.
```
```
❯ rg pg13 --ignore-case -A 2 -B 2 **.c
src/backend/distributed/planner/distributed_planner.c
1838-       if (targetAppendRelInfo != NULL)
1839-       {
1840:           /* postgres deletes translated_vars after pg13, hence we deep copy them here */
1841-           Node *targetNode = NULL;
1842-           foreach_ptr(targetNode, targetAppendRelInfo->translated_vars)

src/backend/distributed/commands/sequence.c
201-             * GetSequencesOwnedByColumn macro and pick initial oid from the
202-             * list instead of using getOwnedSequence. This is both because
203:             * getOwnedSequence is removed in pg13 and is also because it
204-             * errors out if column does not have any sequences.
205-             */

src/backend/distributed/commands/table.c
754-         * This is mainly because postgres started to issue ALTER TABLE commands
755-         * for some set of objects that are defined via CREATE TABLE commands as
756:         * of pg13. However, citus already has a separate logic for CREATE TABLE
757-         * commands.
758-         *
```